### PR TITLE
Readmes and docs

### LINKED ISSUE #4: Add a top-level documentation
* Add a `README.md` to the root of the framework
* Add `.github` folder w/ CONTRIBUTING.md and other templates

### PR DESCRIPTION
**Pull request by smhigley**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [ ] All code has been formatted with [`prettier`](https://prettier.io/) as per the [readme code style guidelines](./../#code-style)
* [ ] Unit or Functional tests are included in the PR

**Description:**

Resolves #4 

It might be worth checking if the packages under Support and Future Packages are still relevant.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+### Dojo CLI
+
+* [`dojo/cli`](https://github.com/dojo/cli) - Command Line Tooling for Dojo Applications
+* [`dojo/cli-build-app`](https://github.com/dojo/cli-build-app) - A CLI command for building Dojo applications
+* [`dojo/cli-build-widget`](https://github.com/dojo/cli-build-widget) - A CLI command for building widgets
+* [`dojo/cli-create-app`](https://github.com/dojo/cli-create-app) - Command for creating application boilerplates
+* [`dojo/cli-create-theme`](https://github.com/dojo/cli-create-theme) - Command for scaffolding a widget theme
+* [`dojo/cli-create-widget`](https://github.com/dojo/cli-create-widget) - Command for creating a widget template and all associated boilerplate
+* [`dojo/cli-export-project`](https://github.com/dojo/cli-export-project) - Command for exporting project bundles
```
Comment (matt-gadd) [inline README.md]: this can be removed (cli-export-project is deprecated)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+### Future packages
+
+We also have packages that are still in their planning stages and as such might not be part of the initial Dojo release.
+
+* [`dojo/dgrid`](https://github.com/dojo/dgrid) - Reactive extensive grid for Dojo
+* [`dojo/crypto`](https://github.com/dojo/crypto) - A set of libraries to deal with cryptography
+* [`dojo/dataviz`](https://github.com/dojo/dataviz) - A data visualization library for Dojo
+
+### Deprecated Packages
```
Comment (matt-gadd) [inline README.md]: perhaps we can ditch this now too.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+* [`dojo/cli`](https://github.com/dojo/cli) - Command Line Tooling for Dojo Applications
+* [`dojo/cli-build-app`](https://github.com/dojo/cli-build-app) - A CLI command for building Dojo applications
+* [`dojo/cli-build-widget`](https://github.com/dojo/cli-build-widget) - A CLI command for building widgets
+* [`dojo/cli-create-app`](https://github.com/dojo/cli-create-app) - Command for creating application boilerplates
+* [`dojo/cli-create-theme`](https://github.com/dojo/cli-create-theme) - Command for scaffolding a widget theme
+* [`dojo/cli-create-widget`](https://github.com/dojo/cli-create-widget) - Command for creating a widget template and all associated boilerplate
+* [`dojo/cli-export-project`](https://github.com/dojo/cli-export-project) - Command for exporting project bundles
+* [`dojo/cli-test-intern`](https://github.com/dojo/cli-test-intern) - Command for testing projects with [Intern](https://theintern.github.io)
+
+### Future packages
```
Comment (matt-gadd) [inline README.md]: I think we can ditch this too

File: README.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
+# @dojo/framework
```
Comment (matt-gadd) [inline README.md]: scrap my comments on this, we can fix this up in a follow up pr before releasing.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+* [`dojo/cli`](https://github.com/dojo/cli) - Command Line Tooling for Dojo Applications
+* [`dojo/cli-build-app`](https://github.com/dojo/cli-build-app) - A CLI command for building Dojo applications
+* [`dojo/cli-build-widget`](https://github.com/dojo/cli-build-widget) - A CLI command for building widgets
+* [`dojo/cli-create-app`](https://github.com/dojo/cli-create-app) - Command for creating application boilerplates
+* [`dojo/cli-create-theme`](https://github.com/dojo/cli-create-theme) - Command for scaffolding a widget theme
+* [`dojo/cli-create-widget`](https://github.com/dojo/cli-create-widget) - Command for creating a widget template and all associated boilerplate
+* [`dojo/cli-test-intern`](https://github.com/dojo/cli-test-intern) - Command for testing projects with [Intern](https://theintern.github.io)
+
+### Support Packages
```
Comment (matt-gadd) [inline README.md]: we can remove all of thse i think.

File: src/core/README.md [hunk 6]
```diff
@@ -138,15 +138,15 @@ See [LICENSE](LICENSE) for details.
 
 <!-- doc-viewer-config
 {
-	"api": "docs/api.json",
+	"api": "../../docs/core/api.json",
```
Comment (nicknisi) [inline src/core/README.md]: Can we remove the relative paths to these files and to the pages? Relative to the top-level of the repository would be consistent with how the other repos do it.

**Pull request merged**